vector/src/main/java/im/vector/app/features/link/LinkHandlerActivity.kt

```diff
@@ -62,9 +68,45 @@ class LinkHandlerActivity : VectorBaseActivity() {
                 startLoginActivity(uri)
             }
         } else {
-            // Other link are not yet handled, but should not comes here (manifest configuration error?)
-            Timber.w("Unable to handle this uir: $uri")
-            finish()
+            if (!sessionHolder.hasActiveSession()) {
+                startLoginActivity(uri)
+                finish()
+            } else {
+                convertUrlToPermalink(uri.toString())?.let { permalink ->
+                    startPermalinkHandler(permalink)
+                } ?: run {
+                    // Other link are not yet handled, but should not comes here (manifest configuration error?)
+                    Timber.w("Unable to handle this uir: $uri")
```

Comment:
There is a typo in the text: `uir` (I know it is not new :) )

Reply:
Fixed.